index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-underscore-dangle, prefer-promise-reject-errors */
```

Comment:
why relax this ? can't we modify the Promise.reject input?

Reply:
updated